pkg/cmd/pipelinerun/logs.go

```diff
@@ -33,6 +33,7 @@ type LogOptions struct {
 	Stream          *cli.Stream
 	Streamer        stream.NewStreamerFunc
 	Tasks           []string
+	Limit           int
```

Comment:
are we using Limit in pipeline run logs as well?
if not then we can limit its scope to local logOptions of pipeline logs

Reply:
No, you're right. It's not needed. I added to `runLogOption` in `pipeline.logs` when first getting started on this thinking it would be, but I will remove it.

Reply:
Removed. Thanks for pointing this out.